packages/fx-core/src/component/driver/aad/utility/aadAppClient.ts

```diff
@@ -99,6 +100,53 @@ export class AadAppClient {
     });
   }
 
+  public async getOwners(objectId: string): Promise<AadOwner[] | undefined> {
+    // const response = await instance.get(`${baseUrl}/applications/${objectId}/owners`);
+    const response = await this.axios.get(`applications/${objectId}/owners`, {
+      "axios-retry": {
+        retries: this.retryNumber,
+        retryDelay: axiosRetry.exponentialDelay,
+        retryCondition: (error) =>
+          axiosRetry.isNetworkError(error) ||
+          axiosRetry.isRetryableError(error) ||
+          this.is404Error(error), // also retry 404 error since AAD need sometime to sync created AAD app data
+      },
+    });
+
+    const aadOwners: AadOwner[] = [];
+    if (response && response.data && response.data.value) {
```

Comment:
Is it possible that response, response.data (and response.data.value) can be undefined per MS Graph API document? If not, maybe we can remove the check?

Reply:
We add this check because we met with undefined response.data.value used to return undefined when owner is empty for an app. I tried this api again and now it seems will return empty array. I will update here.